src/modules/cpu/kernel/normalize_generic.hpp

```diff
@@ -0,0 +1,513 @@
+        srcPtrTemp += srcDescPtr->strides[1];
+        dstPtrTemp += dstDescPtr->strides[1];
+    }
+}
+
+void normalize_2D_tensor_avx_axis2(Rpp32f *srcPtr, RpptGenericDescPtr srcDescPtr, Rpp32f *dstPtr, RpptGenericDescPtr dstDescPtr,
+                                   Rpp32f *meanPtr, Rpp32f *invStdDevPtr, Rpp32f shift, Rpp32u *dims, Rpp32u *paramStride)
+{
+    Rpp32f *srcPtrTemp = srcPtr;
+    Rpp32f *dstPtrTemp = dstPtr;
+    Rpp32s paramIdx = 0;
+    Rpp32u vectorIncrement = 8;
+    Rpp32u bufferLength = dims[1];
+    Rpp32u alignedLength = (bufferLength / 8) * 8;
+    Rpp32u numRows = dims[0];
+
+    __m256 pShift = _mm256_set1_ps(shift);
+    for(Rpp32u i = 0; i < numRows; i++)
+    {
+        Rpp32f *srcPtrTempRow = srcPtrTemp + i * srcDescPtr->strides[1];
+        Rpp32f *dstPtrTempRow = dstPtrTemp + i * dstDescPtr->strides[1];
+
+        // set mean and stddev
+        Rpp32f mean = meanPtr[i];
+        Rpp32f invStdDev = invStdDevPtr[i];
+        __m256 pMean, pInvStdDev;
+        pMean = _mm256_set1_ps(mean);
+        pInvStdDev = _mm256_set1_ps(invStdDev);
+
+        Rpp32u vectorLoopCount = 0;
+        for(; vectorLoopCount < alignedLength ; vectorLoopCount += 8)
+        {
+            __m256 pSrc = _mm256_loadu_ps(srcPtrTempRow);
+            __m256 pDst = _mm256_add_ps(_mm256_mul_ps(_mm256_sub_ps(pSrc, pMean), pInvStdDev), pShift);
+            _mm256_storeu_ps(dstPtrTempRow, pDst);
+            srcPtrTempRow += 8;
+            dstPtrTempRow += 8;
+        }
+        for(; vectorLoopCount < dims[1] ; vectorLoopCount += 8)
+             *dstPtrTempRow++ = (*srcPtrTempRow++ - mean) * invStdDev + shift;
+    }
+}
+
+RppStatus normalize_generic_f32_f32_host_tensor(Rpp32f *srcPtr,
+                                                RpptGenericDescPtr srcGenericDescPtr,
+                                                Rpp32f *dstPtr,
+                                                RpptGenericDescPtr dstGenericDescPtr,
+                                                Rpp32u axis_mask,
+                                                Rpp32f *meanTensor,
+                                                Rpp32f *stdDevTensor,
+                                                Rpp32u computeMean,
+                                                Rpp32u computeStddev,
+                                                Rpp32f scale,
+                                                Rpp32f shift,
+                                                Rpp32u *roiTensor,
+                                                RppLayoutParams layoutParams,
+                                                rpp::Handle& handle)
+{
+	Rpp32u numThreads = handle.GetNumThreads();
+    Rpp32u nDim = srcGenericDescPtr->numDims;
+    Rpp32u batchSize = dstGenericDescPtr->dims[0];
+
+    omp_set_dynamic(0);
+#pragma omp parallel for num_threads(numThreads)
+	for(int batchCount = 0; batchCount < batchSize; batchCount++)
```

Comment:
Remove the additional spaces for loop
Please make  it as a practice to write code with indentations during development itself, so that time can be reduced later for cleanups. Applicable to all such alignment/indentation issues in this file

Reply:
alignment seem fine even here when checked with VS Code